docs/resources/samples/filter-table-get-visible-range.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Filter Excel table and get visible range'
+description: 'Learn how to use Office Scripts to filter an Excel table and get the visible range as an array of objects.'
+ms.date: 03/16/2021
+localization_priority: Normal
+---
+
+# Filter Excel table and get visible range as a JSON object
+
+This sample filters an Excel table and returns the visible range as a JSON object. This JSON could be provided to a Power Automate flow as part of a larger solution.
+
+## Example scenario
+
+* Apply a filter to a table column.
+* Extract the visible range after filtering.
+* Assemble and return an object with a [specific JSON structure](#sample-json).
+
+## Office Script
+
+The following script filters a table and gets the visible range.
+
+Download the sample file <a href="table-filter.xlsx">table-filter.xlsx</a> and use it with this script to try it out yourself!
+
+```TypeScript
```

Comment:
add code comments

Reply:
Yeah, I had received this feedback before. We're tracking with this item: 4958338